src/content/en/fundamentals/performance/rail.md

```diff
@@ -108,8 +119,7 @@ Working in 50ms blocks allows the task to finish while still ensuring instant re
 
 ## Load: deliver content under 1000ms
 
-Load your site in under 1 second.
-If you don't, your user's attention wanders,
+Load your site in under 1 second. If you don't, user attention wanders,
```

Comment:
Once we've nailed down our definition of Time To Interactive, I agree it would be good to use TTI here. Until then, this seems good enough to me.